dpnp/tests/test_linalg.py

```diff
@@ -2785,6 +2775,12 @@ def test_slogdet_strides(self):
         assert_allclose(sign_result, sign_expected)
         assert_allclose(logdet_result, logdet_expected)
 
+    # TODO: remove skipif when Intel MKL 2025.2 is released
+    # Skip running on cpu because dpnp uses _getrf_batch only on cpu.
```

Comment:
```suggestion
    # Skip running on CPU because dpnp uses _getrf_batch only on CPU.
```
The comment is quite unclear why we are skipping on CPU only, because `_getrf_batch` is used by `_batched_inv` and `_lu_factor` and there is no check that it is only when CPU device. OR do I miss something?

Reply:
`dpnp.linalg.slogdet/det` call `getf_batch` through `_lu_factor()` which uses `getf_batch` only on CPU because of slow performance on GPU. That is why this condition was added here

`_batched_inv` calls `getf_batch` directly and `getf_batch` is used on GPU and CPU.

I have updated the comment and I will investigate the performance of `gerf_batch` on GPU again to consistent logic( In the next PR).